From: Bertrand Lorentz <bertrand.lorentz@gmail.com>
Date: Sat, 12 Sep 2015 21:59:33 +0800
Subject: build: Remove obsolete check for gnome-desktop-2.0 >= 2.28

The check was just used to put the right name in the .desktop file. We
keep the check for MeeGo to still do the right thing there.

From: commit:ed39c120a1aa604d0b5e324580a044715c033a17
---
 configure.ac | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 5e272ba..7926526 100644
--- a/configure.ac
+++ b/configure.ac
@@ -69,12 +69,9 @@ SHAMROCK_FIND_MONO_2_0_COMPILER
 SHAMROCK_FIND_MONO_RUNTIME
 SHAMROCK_CHECK_MONO_2_0_GAC_ASSEMBLIES([Mono.Cairo Mono.Posix ICSharpCode.SharpZipLib])
 
-dnl Check for GNOME >= 2.28 to see if we should remove
-dnl 'Media Player' from the app Name, bgo#596242, except
-dnl in MeeGo where GNOME >= 2.28, but we want the full name
-PKG_CHECK_MODULES(GNOME_2_28, gnome-desktop-2.0 >= 2.28, HAVE_GNOME_2_28=yes, HAVE_GNOME_2_28=no)
+dnl In MeeGo we want the full name, but not in GNOME 3 (see bgo#596242)
 PKG_CHECK_MODULES(MEEGO_PANEL, meego-panel, HAVE_MEEGO_PANEL=yes, HAVE_MEEGO_PANEL=no)
-if test "x$HAVE_GNOME_2_28" = "xno" -o "x$HAVE_MEEGO_PANEL" = "xyes"; then
+if "x$HAVE_MEEGO_PANEL" = "xyes"; then
 	BANSHEE_DESKTOP_NAME="_Name=Banshee Media Player"
 else
 	BANSHEE_DESKTOP_NAME="_Name=Banshee"
